Project/core/src/main/java/View/buildingBlocks/VisualCell.java

```diff
@@ -173,7 +177,10 @@ private void updateTokens() {
                 tokens.add(brains[0]);
             } else if (token instanceof BrainFact) {
                 tokens.add(brains[1]);
-            } else {
+            }else if (token instanceof MindSlip) {
+                tokens.add(mindslip);
```

Comment:
similar as before, i like the implementation! its good, but according to the rules we are implementing we dont need this right now

Reply:
Deleted it now.